Test failures locally

Trying to do a release and get this failure:
```
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Building webtau-report-testing 1.17
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO]
[INFO] [INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ webtau-report-testing ---
[INFO] [INFO]
[INFO] [INFO] --- webtau-maven-plugin:1.17:run (default) @ webtau-report-testing ---
[INFO] [INFO] test files:
[INFO]     scenarios/crudOverview.groovy
[INFO]     scenarios/crudSeparatedOverview.groovy
[INFO]     scenarios/junit5Report.groovy
[INFO]   workingDir: /Users/karol/OSSDev/webtau/webtau-report-testing/src/test/groovy // from command line argument
[INFO]      docPath: ../../../target/doc-artifacts/reports                            // from config file
[INFO]  windowWidth: 1200                                                             // from config file
[INFO] windowHeight: 920                                                              // from config file
[INFO]
[INFO] http calls (crudOverview.groovy)
[INFO] preparing chrome WebDriver
[INFO] [INFO] Using chromedriver 79.0.3945.36 (since Google Chrome 79 is installed in your machine)
[INFO] [INFO] Exporting webdriver.chrome.driver as /Users/karol/.m2/repository/webdriver/chromedriver/mac64/79.0.3945.36/chromedriver
[INFO] Starting ChromeDriver 79.0.3945.36 (3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}) on port 37913
[INFO] Only local connections are allowed.
[INFO] Please protect ports used by ChromeDriver and related test frameworks to prevent access by malicious code.
[INFO] [1578091156.837][WARNING]: FromSockAddr failed on netmask
[INFO] [1578091156.837][WARNING]: FromSockAddr failed on netmask
[INFO] > opening file:///Users/karol/OSSDev/webtau/webtau-report-testing/src/test/groovy/../../../../webtau-feature-testing/target/classes/scenarios/rest/springboot/customerCrud-webtau-report.html
[INFO] . opened file:///Users/karol/OSSDev/webtau/webtau-report-testing/src/test/groovy/../../../../webtau-feature-testing/target/classes/scenarios/rest/springboot/customerCrud-webtau-report.html (251ms)
[INFO] > waiting to by css .navigation-entry .label , element(s) with text CRUD operations for customer to be visible
[INFO] . by css .navigation-entry .label , element(s) with text CRUD operations for customer is visible (78ms)
[INFO] > clicking by css .navigation-entry .label , element(s) with text CRUD operations for customer
[INFO] . clicked by css .navigation-entry .label , element(s) with text CRUD operations for customer (57ms)
[INFO] > clicking by css .tab-selection .tab-name , element(s) with text HTTP calls
[INFO] . clicked by css .tab-selection .tab-name , element(s) with text HTTP calls (55ms)
[INFO] > waiting to by css .test-http-call to be visible
[INFO] . by css .test-http-call is visible (29ms)
[INFO] > clicking by css .test-http-call , element number 2 , nested find by css .collapse-toggle
[INFO] . clicked by css .test-http-call , element number 2 , nested find by css .collapse-toggle (50ms)
[INFO] [.] http calls (crudOverview.groovy)
[INFO]
[INFO] summary view (crudSeparatedOverview.groovy)
[INFO] > opening file:///Users/karol/OSSDev/webtau/webtau-report-testing/src/test/groovy/../../../../webtau-feature-testing/target/classes/scenarios/rest/springboot/customerCrudSeparatedMissingMethod-webtau-report.html
[INFO] . opened file:///Users/karol/OSSDev/webtau/webtau-report-testing/src/test/groovy/../../../../webtau-feature-testing/target/classes/scenarios/rest/springboot/customerCrudSeparatedMissingMethod-webtau-report.html (67ms)
[INFO] [.] summary view (crudSeparatedOverview.groovy)
[INFO]
[INFO] customer update test summary (crudSeparatedOverview.groovy)
[INFO] > waiting to by css .navigation-entry .label , element(s) with text customer update to be visible
[INFO] . by css .navigation-entry .label , element(s) with text customer update is visible (31ms)
[INFO] > clicking by css .navigation-entry .label , element(s) with text customer update
[INFO] . clicked by css .navigation-entry .label , element(s) with text customer update (51ms)
[INFO] [.] customer update test summary (crudSeparatedOverview.groovy)
[INFO]
[INFO] customer update http calls (crudSeparatedOverview.groovy)
[INFO] > opening file:///Users/karol/OSSDev/webtau/webtau-report-testing/src/test/groovy/../../../../webtau-feature-testing/target/classes/scenarios/rest/springboot/customerCrudSeparated-webtau-report.html
[INFO] . opened file:///Users/karol/OSSDev/webtau/webtau-report-testing/src/test/groovy/../../../../webtau-feature-testing/target/classes/scenarios/rest/springboot/customerCrudSeparated-webtau-report.html (60ms)
[INFO] > waiting to by css .navigation-entry .label , element(s) with text customer update to be visible
[INFO] . by css .navigation-entry .label , element(s) with text customer update is visible (143ms)
[INFO] > clicking by css .navigation-entry .label , element(s) with text customer update
[INFO] . clicked by css .navigation-entry .label , element(s) with text customer update (52ms)
[INFO] > clicking by css .tab-selection .tab-name , element(s) with text HTTP calls
[INFO] . clicked by css .tab-selection .tab-name , element(s) with text HTTP calls (51ms)
[INFO] > waiting to by css .test-http-call to be visible
[INFO] . by css .test-http-call is visible (20ms)
[INFO] > clicking by css .test-http-call , element number 2 , nested find by css .collapse-toggle
[INFO] . clicked by css .test-http-call , element number 2 , nested find by css .collapse-toggle (52ms)
[INFO] [.] customer update http calls (crudSeparatedOverview.groovy)
[INFO]
[INFO] test lists (junit5Report.groovy)
[INFO] > opening file:///Users/karol/OSSDev/webtau/webtau-report-testing/src/test/groovy/../../../../webtau-junit5-examples/webtau.report.html
[INFO] . opened file:///Users/karol/OSSDev/webtau/webtau-report-testing/src/test/groovy/../../../../webtau-junit5-examples/webtau.report.html (60ms)
[INFO] > expecting by css .group-of-tests .navigation-entry-group-label to contain "customer query"
[INFO] X failed expecting by css .group-of-tests .navigation-entry-group-label to contain "customer query" :
[INFO]     [value] expect to contain "customer query"
[INFO]     [value]:
[INFO] [x] test lists (junit5Report.groovy)
[INFO]
[INFO] Failed tests:
[INFO] [x] test lists scenarios/junit5Report.groovy
[INFO] java.lang.AssertionError:
[INFO] [value] expect to contain "customer query"
[INFO] [value]:
[INFO] 	at scenarios.junit5Report$_run_closure1.doCall(junit5Report.groovy:9)
[INFO] 	at scenarios.junit5Report$_run_closure1.doCall(junit5Report.groovy)
[INFO] 	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
[INFO] 	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
[INFO] 	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
[INFO] 	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
[INFO] 	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
[INFO] 	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
[INFO] 	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
[INFO] 	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
[INFO] 	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:309)
[INFO] 	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:194)
[INFO] 	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:107)
[INFO] 	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:955)
[INFO] 	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:290)
[INFO] 	at org.apache.maven.cli.MavenCli.main(MavenCli.java:194)
[INFO] 	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
[INFO] 	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
[INFO] 	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
[INFO] 	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
[INFO]
[INFO] Total: 5,  Passed: 4,  Skipped: 0,  Failed: 1,  Errored: 0
[INFO] report is generated:  /Users/karol/OSSDev/webtau/webtau-report-testing/src/test/groovy/webtau.report.html
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Reactor Summary:
[INFO] [INFO]
[INFO] [INFO] WebTau ............................................. SUCCESS [  0.562 s]
[INFO] [INFO] webtau-utils-testing-data .......................... SUCCESS [  0.590 s]
[INFO] [INFO] webtau-utils ....................................... SUCCESS [  3.710 s]
[INFO] [INFO] webtau-console ..................................... SUCCESS [  2.049 s]
[INFO] [INFO] webtau-core ........................................ SUCCESS [  6.266 s]
[INFO] [INFO] webtau-groovy-ast .................................. SUCCESS [  2.543 s]
[INFO] [INFO] webtau-core-groovy ................................. SUCCESS [  4.387 s]
[INFO] [INFO] webtau-config ...................................... SUCCESS [  2.454 s]
[INFO] [INFO] webtau-reactjs ..................................... SUCCESS [ 19.260 s]
[INFO] [INFO] webtau-report ...................................... SUCCESS [  2.381 s]
[INFO] [INFO] webtau-standalone-runner-groovy .................... SUCCESS [  4.649 s]
[INFO] [INFO] webtau-cache ....................................... SUCCESS [  2.367 s]
[INFO] [INFO] webtau-browser ..................................... SUCCESS [  3.597 s]
[INFO] [INFO] webtau-test-server ................................. SUCCESS [  1.146 s]
[INFO] [INFO] webtau-http ........................................ SUCCESS [  4.841 s]
[INFO] [INFO] webtau-http-groovy ................................. SUCCESS [  5.973 s]
[INFO] [INFO] webtau-open-api .................................... SUCCESS [  4.005 s]
[INFO] [INFO] webtau-pdf ......................................... SUCCESS [ 29.841 s]
[INFO] [INFO] webtau-json-schema ................................. SUCCESS [  02:01 h]
[INFO] [INFO] webtau-data ........................................ SUCCESS [  3.006 s]
[INFO] [INFO] webtau-cli ......................................... SUCCESS [  2.904 s]
[INFO] [INFO] webtau-cli-groovy .................................. SUCCESS [  4.312 s]
[INFO] [INFO] webtau ............................................. SUCCESS [  1.189 s]
[INFO] [INFO] webtau-groovy ...................................... SUCCESS [  8.032 s]
[INFO] [INFO] webtau-java-runner ................................. SUCCESS [  0.797 s]
[INFO] [INFO] webtau-junit5 ...................................... SUCCESS [  0.978 s]
[INFO] [INFO] webtau-junit5-groovy ............................... SUCCESS [  1.804 s]
[INFO] [INFO] webtau-junit-like-examples-common .................. SUCCESS [  0.840 s]
[INFO] [INFO] webtau-testapp ..................................... SUCCESS [  2.786 s]
[INFO] [INFO] webtau-junit5-examples ............................. SUCCESS [ 18.959 s]
[INFO] [INFO] webtau-junit4 ...................................... SUCCESS [  0.879 s]
[INFO] [INFO] webtau-feature-testing ............................. SUCCESS [ 38.294 s]
[INFO] [INFO] webtau-junit4-examples ............................. SUCCESS [ 11.576 s]
[INFO] [INFO] webtau-maven-plugin ................................ SUCCESS [  8.771 s]
[INFO] [INFO] webtau-report-testing .............................. FAILURE [  8.531 s]
[INFO] [INFO] webtau-cli-testing ................................. SKIPPED
[INFO] [INFO] webtau-docs ........................................ SKIPPED
[INFO] [INFO] webtau-maven-plugin-test ........................... SKIPPED
[INFO] [INFO] webtau-maven-plugin-test-skip ...................... SKIPPED
[INFO] [INFO] webtau-shaded ...................................... SKIPPED
[INFO] [INFO] webtau-dist ........................................ SKIPPED
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] BUILD FAILURE
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Total time: 02:04 h
[INFO] [INFO] Finished at: 2020-01-03T22:39:23Z
[INFO] [INFO] Final Memory: 341M/2937M
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [ERROR] Failed to execute goal com.twosigma.webtau:webtau-maven-plugin:1.17:run (default) on project webtau-report-testing: Execution default of goal com.twosigma.webtau:webtau-maven-plugin:1.17:run failed.: UndeclaredThrowableException: tests failure: check tests output -> [Help 1]
```

--- a/webtau-report-testing/src/test/groovy/pages/Report.groovy
+++ b/webtau-report-testing/src/test/groovy/pages/Report.groovy
@@ -9,11 +9,11 @@ class Report {
     def groupNames = $(".group-of-tests .navigation-entry-group-label")
 
     def openGroovyStandaloneReport(String reportName) {
-        browser.open(ReportLocation.groovyFeatureTestingFullUrl(reportName))
+        openReportFile(ReportLocation.groovyFeatureTestingFullUrl(reportName))
     }
 
     def openJunit5ExampleReport() {
-        browser.open(ReportLocation.javaJunit5FullUrl('webtau.report.html'))
+        openReportFile(ReportLocation.javaJunit5FullUrl('webtau.report.html'))
     }
 
     def selectTest(String testName) {
@@ -30,5 +30,10 @@ class Report {
         def httpCalls = $(".test-http-call")
         httpCalls.waitTo beVisible()
         httpCalls.get(number).find(".collapse-toggle").click()
+    }
+
+    private static def openReportFile(String fileName) {
+        browser.open(fileName)
+        $(".status-filter-area").waitTo beVisible()
     }
 }

